agent/acs/model/api/api-2.json

```diff
@@ -551,6 +582,13 @@
         "asm"
       ]
     },
+    "SecretTarget":{
```

Comment:
(May be I'm clueless because I don't know the agent that well) Why are things related to secrets added in this?

Reply:
i think last time this field was manually added to api.go.. so if we don't add this field to api-2.json, generating api.go from api-2.json will cause this field be deleted from api.go..

Reply:
Oh I see, I should've guessed this file was auto-generated